### PR TITLE
Remove warnings about comments beta

### PR DESCRIPTION
**Pull request by nimeshnayaju**
### Description

Preparation for open beta: Removes warnings about comments beta when using comment hooks like `useThreads`, `useCreateThread`, etc.

### CODE REVIEW

File: packages/liveblocks-react-comments/README.md [hunk 1]
```diff
@@ -27,9 +27,10 @@ and scale collaborative applications with zero configuration, no maintenance
 required.
 
 > [!IMPORTANT]  
-> Comments is currently in private beta and is only available to a select group
-> of users. If you want to participate in the beta,
-> [sign up for the waitlist](https://forms.gle/xgp54aTMsoH8hrjU9).
+> Comments is currently in beta.If you have any questions or feedback, please
+> reach out to us via the dedicated Discord channel,
+> [#comments-beta](https://discord.com/channels/913109211746009108/1126614905160749097).
+> We’d love to hear from you.
```
Comment (marcbouchenoire) [inline packages/liveblocks-react-comments/README.md:33]: I think we can remove this one entirely.

**Review by marcbouchenoire — CHANGES_REQUESTED** [review]
Nice! Let's keep this open until we're ready to release.

File: packages/liveblocks-react-comments/README.md (hunk 1 above)
Comment (nimeshnayaju) [reply packages/liveblocks-react-comments/README.md:33]: Sorry for replying late. I was focused on other things last week. Were you suggesting to remove the whole text from the README or just the last line `We'd love to hear you` from the text?

Comment (marcbouchenoire) [reply packages/liveblocks-react-comments/README.md:33]: The whole text, I think we say it enough times on the site and the docs.

Comment (nimeshnayaju) [reply packages/liveblocks-react-comments/README.md]: Thanks! and done

**Review by marcbouchenoire — APPROVED** [review]

**Pull request merged**